Tests/scripts/hook_validations/docker.py

```diff
@@ -0,0 +1,240 @@
+        if not self.docker_image_tag and not self.docker_image_name and not self.docker_image_latest_tag:
+            self.is_latest_tag = False
+        else:
+            # Case of a modified file with version >= 5.0.0
+            if self.is_modified_file and server_version_compare(self.from_version, '5.0.0') >= 0:
+                # If docker image name are different and if the docker image isn't the default one
+                if self.docker_image_latest_tag != self.docker_image_tag and not \
+                        'demisto/python:1.3-alpine' == '{}:{}'.format(self.docker_image_name, self.docker_image_tag):
+                    self.is_latest_tag = False
+            # Case of an added file
+            elif not self.is_modified_file:
+                if self.docker_image_latest_tag != self.docker_image_tag:
+                    self.is_latest_tag = False
+
+        if not self.is_latest_tag:
+            print_error('The docker image tag is not the latest, please update it.')
+        return self.is_latest_tag
+
+    @staticmethod
+    def parse_www_auth(www_auth):
+        """Parse realm and service from www-authenticate string of the form:
+        Bearer realm="https://auth.docker.io/token",service="registry.docker.io"
+
+        :param www_auth: www-authenticate header value
+        :type www_auth: string
+        """
+        match = re.match(r'.*realm="(.+)",service="(.+)".*', www_auth, re.IGNORECASE)
+        if not match:
+            return []
+        return [match.group(1), match.group(2)]
+
+    @staticmethod
+    def docker_auth(image_name, verify_ssl=True, registry=DEFAULT_REGISTRY):
+        """
+        Authenticate to the docker service. Return an authentication token if authentication is required.
+        """
+        res = requests.get(
+            'https://{}/v2/'.format(registry),
+            headers=ACCEPT_HEADER,
+            timeout=TIMEOUT,
+            verify=verify_ssl
+        )
+        if res.status_code == 401:  # need to authenticate
+            # defaults in case we fail for some reason
+            realm = 'https://auth.docker.io/token'
+            service = 'registry.docker.io'
+            # Should contain header: Www-Authenticate
+            www_auth = res.headers.get('www-authenticate')
+            if www_auth:
+                parse_auth = DockerImageValidator.parse_www_auth(www_auth)
+                if parse_auth and len(parse_auth) == 2:
+                    realm, service = parse_auth
+            res = requests.get(
+                '{}?scope=repository:{}:pull&service={}'.format(realm, image_name, service),
+                headers=ACCEPT_HEADER,
+                timeout=TIMEOUT,
+                verify=verify_ssl
+            )
+            res.raise_for_status()
+            res_json = res.json()
+            return res_json.get('token')
+        else:
+            res.raise_for_status()
+            return None
+
+    @staticmethod
+    def clear_non_numbered_tags(tags):
```

Comment:
```
numeric_tag_regex = re.compile(r'([\d.]+)')
return [tag for tag in tags if numeric_tag_regex.match(tag)]
```

also look at `test_utils:get_last_release_version` you are doing something very similar.